package.json

```diff
@@ -100,13 +102,20 @@
     "write-file-webpack-plugin": "^4.5.1"
   },
   "dependencies": {
-    "@modusoperandi/licit-ui-commands": "^0.1.16",
+    "@babel/plugin-proposal-json-strings": "^7.18.6",
```

Comment:
babel and types should be devDependencies, and the rest should be peerDependnecies.